webui/src/services/okrservice.js

```diff
@@ -214,10 +214,40 @@ export default class OkrService {
     }
 
     shareNotebook(notebookId, errHandler) {
+        const oneDriveETag = notebookId.substr(2);
+        const oneDriveSearchUrl = `me/drive/root/search(q='{${oneDriveETag}}')`;
+        this.graphClient
+            .api(oneDriveSearchUrl)
+            .get()
```

Comment:
It is possible to provide count() ($count odata) parameter and get already calculated length. Nano-optimization :smile: Does not make sense here anyway)

Reply:
Agree )